src/Components/GenreSelector.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+export class GenreSelector extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            selected: parseInt(props.selectedGenre),
```

Comment:
The selected state should be based on the genre name, not the position in the list :)

Reply:
Here you could do something like pass down the genre name as selected property and:
```
selected: props.genreList.find(genre => genre.name === props.selected),
```
Then, use genreList directly in the component. Might not required to store this value in the state.

Reply:
I am passing the ID, and not the position.